src/components/basicComponents/RoadmapMenu.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <li>
     <BaseBox variant="small">
-      <div class="mb-6 flex justify-between">
-        <h2 class="text-lg text-neutral-500">Roadmap</h2>
+      <div class="mb-6 flex justify-between pt-[1.1875rem]">
```

Comment:
I would add some values to config like 2,4,6,8 etc and use it here like: pt-12 or sth :) Usually you shouldn't use custom paddings / margins because they should be a part of ds

Reply:
Same like colors & font sizes

Reply:
Ok apparently I changed it before. Noted for the future :)

Reply:
ok noted and for this branch done, I will correct systematically with next branches / prs